scripts/install_deps.sh

```diff
@@ -165,11 +165,12 @@ case $(uname -s) in
 # Debian
 #------------------------------------------------------------------------------
 
-            Debian)
+            Debian*)
                 #Debian
+                . /etc/os-release
                 install_z3=""
-                case $(lsb_release -cs) in
-                    wheezy)
+                case $VERSION_ID in
```

Comment:
This is a way of doing the same if you don't have `lsb_release` tool (common in docker images).